# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1326396635 18000
#      Thu Jan 12 14:30:35 2012 -0500
# Branch stable
# Node ID 0b0161e56f5a55265bdc5de503feb510507a2f31
# Parent  39c0e0069895417a314dad1049556159816fc0ee
doc: fix incorrect menu

* preface.txi: Delete nonexistent section in menu.

diff --git a/doc/interpreter/preface.txi b/doc/interpreter/preface.txi
--- a/doc/interpreter/preface.txi
+++ b/doc/interpreter/preface.txi
@@ -60,17 +60,16 @@ Everyone is encouraged to share this sof
 terms of the GNU General Public License (@pxref{Copying}).  You are 
 also encouraged to help make Octave more useful by writing and 
 contributing additional functions for it, and by reporting any problems
 you may have.
 
 @menu
 * Acknowledgements::            
 * How You Can Contribute to Octave::  
-* Donations to Support Octave Development::
 * Distribution::                
 @end menu
 
 @node Acknowledgements
 @unnumberedsec Acknowledgements
 @cindex acknowledgements
 @end ifclear
 
